### PR TITLE
Integrate with API SDK node

### PR DESCRIPTION
**Pull request by HarryBosworth**
This pull request integrates the Account Validator web with the Account Validator API by using the API SDK node.

It includes the following changes:
- Added middleware errorHandler and handleErrors function to capture and handle uncaught exceptions.
- Extended the AccountValidationService interface to include a check function to check the status of a previously submitted validation request.
- Created a PendingValidationResult interface to represent the status of a validation request that is still pending, and a mapResponseType function to map the response type from the API to the AccountValidationResult type.
- Added a SynchronousValidator class to continuously check the status of a validation request.
- Added functions to create API clients for public use in api.service.ts.
- Updated the unit tests to reflect the changes.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (modhasanjay) [inline .vscode/launch.json:4]: Is this file part of our source code as it is under .vscode folder or can this be ignored?

File: views/error/service-offline.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Service offline - File a confirmation statement
```
Comment (modhasanjay) [inline views/error/service-offline.html:4]: I see that we are using this offline page using an example from confirmation statement. Could you please customize it with the text relevant for our new test micro-service.

File: .vscode/launch.json (hunk 1 above)
Comment (HarryBosworth) [reply .vscode/launch.json:4]: It's the configuration file for debugging. Not technically part of the code, but useful for anyone who wants to debug, so I left it in. It's similar to the `.idea` foler that Intellj creates

File: views/error/service-offline.html (hunk 2 above)
Comment (HarryBosworth) [reply views/error/service-offline.html:4]: Ah, I missed this reference. I'll Change it now and make another commit.

**Review by modhasanjay — APPROVED** [review]
ok

**Review by modhasanjay — COMMENTED** [review]
ok

**Pull request merged**